test_py/plasticonding/test_development.py

```diff
@@ -101,6 +102,7 @@ def test_measurements_body(self):
         self.assertAlmostEqual(connectivity2_abs, m.extensiveness, 3)
         # self.assertAlmostEqual(connectivity3, m., 3)
         self.assertAlmostEqual(connectivity4, m.branching_modules_count, 3)
+        self.assertAlmostEqual(connectivity5, m.tx_branching_modules_count, 3)
```

Comment:
why not assertEqual?

Reply:
because these are floating point values

Reply:
Why is a counter a floating point value, can you count half branching modules?

Reply:
good point, this is an integer counter :/